src/memory/AmdFlash.cc

```diff
@@ -555,6 +573,35 @@ bool AmdFlash::checkCommandLongReset()
 	return false;
 }
 
+bool AmdFlash::checkCommandStatusRead()
+{
+	static constexpr std::array<uint8_t, 3> cmdSeq = {0x70};
+	if (chip.misc.statusCommand && partialMatch(cmdSeq)) {
+		setState(State::STATUS);
+	}
+	return false;
+}
+
+bool AmdFlash::checkCommandStatusClear()
+{
+	static constexpr std::array<uint8_t, 3> cmdSeq = {0x71};
+	if (chip.misc.statusCommand && partialMatch(cmdSeq)) {
+		softReset();
+	}
+	return false;
+}
+
+bool AmdFlash::checkCommandContinuityCheck()
+{
+	if (chip.misc.continuityCommand && cmd[0].addr == 0x5554AB && cmd[0].value == 0xFF) {
+		if (cmdIdx < 2) return true;
+		if (cmdIdx == 2 && cmd[1].addr == 0x2AAB54 && cmd[1].value == 0x00) {
```

Comment:
Detail. I only notice this now. I'm not sure, but is it clearer to write this like:
`if (cmdIdx == 2 && cmd[1] == AmdCmd{0x2AAB54, 0x00}) {`
But maybe `AmdCmd` isn't the best name. Maybe something like `AddrAndValue`?

2 lines above something similar could be done. And possibly in various other places in the code (I didn't check).